gulpfile.js

```diff
@@ -1,5 +1,4 @@
-/* global require */
```

Comment:
Yup, meant to remove it.  Should be superseded by `/* jshint node: true */`, and since Travis didn't barf on the distribution-building files, we should be good.